.github/workflows/cypress-test.yml

```diff
@@ -10,14 +10,15 @@ env:
   OPENSEARCH_SNAPSHOT_CMD: 'node ../scripts/opensearch snapshot'
   SPEC: 'cypress/integration/plugins/security-dashboards-plugin/aggregation_view.js,'
   PLUGIN_NAME: opensearch-security
+  OPENSEARCH_INITIAL_ADMIN_PASSWORD: myStrongPassword123!
 
 jobs:
-  tests:
+  cypress-tests:
     name: Run Cypress tests
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-latest , windows-latest ]
+        os: [ubuntu-latest]
```

Comment:
Why remove windows support?

Reply:
Was trying to pull some stability changes from @cwperks' PR, will merge main into this to remove the change.

Reply:
👍